.github/workflows/security-build.yml

```diff
@@ -0,0 +1,36 @@
+name: Security Build
+on:
+  push:
+  workflow_dispatch: {}
+
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
```

Comment:
Do we need linting check here, since we already have it run as a "standalone" GH action:
https://github.com/0xPolygon/cdk-validium-node/blob/develop/.github/workflows/lint.yml.

Also since check is failing (https://github.com/0xPolygon/cdk-validium-node/actions/runs/8113861811/job/22178167788?pr=94), it seems to me like checkout code part is lacking here. Can you take a look into it, in case we need linting check here as well?